project/plugins.sbt

```diff
@@ -20,3 +20,6 @@ addSbtPlugin("com.github.ehsanyou" % "sbt-docker-compose" % "67284e73-envvars-2m
 // depend directly on the patched version see https://github.com/akka/alpakka/issues/1388
 addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.9.2+10-148ba0ff")
 resolvers += Resolver.bintrayIvyRepo("2m", "sbt-plugins")
+
+addSbtPlugin("net.aichler" % "sbt-jupiter-interface" % "0.8.0")
+resolvers += Resolver.jcenterRepo
```

Comment:
When using `coursier` I need this resolver also in `build.sbt`

Reply:
Sorry, forgot to add it there.